### DISCUSSION TITLE
How to add a radius limit while circularising the closed loop

### DISCUSSION THREAD

**Discussion by anithaswain-tomtomexternal** [openstreetmap/iD#10041] [Ideas]
can any one please share how to add a radius limit while circularising the closed loop

**Comment by k-yle** [comment]
This is not possible currently, but #9732 would allow you to place a node at the center and tag it with a `diameter` or `radius`. You could then adjust your circle until it lines up.

**Comment by tordans** [comment]
Another Workaround would be to draw a temporary line and use the measurement panel to see it width and then draw a triangle using this line as reference and make it circular


1. <img width="295" alt="image" src="https://github.com/user-attachments/assets/41a1928e-9f23-41a7-8123-c65658d97693">
2. <img width="554" alt="image" src="https://github.com/user-attachments/assets/360205c0-8c6d-41ee-8343-37e2af447a71">
